package.json

```diff
@@ -3,27 +3,30 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@craco/craco": "^6.1.2",
+    "@babel/core": "^7.9.0",
```

Comment:
you shouldn't need this or the other babel plugin, you need to eject from CRA in order to mess with the babel config

Reply:
instead of looking at the React install guide for twin.macro, look at the create-react-app install guide for twin.macro: https://github.com/ben-rogerson/twin.examples/tree/master/cra-styled-components